Style identity and balance sections in sidebar

MiZiet opened the pull request:
If applied these changes will:
- style identity section in sidebar,
- style balances section in sidebar.

MiZiet commented on packages/page-accounts/src/Sidebar/Sidebar.spec.tsx:
```diff
@@ -0,0 +1,128 @@
+const aliceAddress = '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY';
+const bobAddress = '5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty';
+
+describe('Sidebar, subs list', () => {
```
Once #6049 is merged we can refactor these tests.

krzysztof-jelski commented on packages/page-accounts/src/Sidebar/Identity.tsx:
```diff
@@ -24,20 +26,17 @@ function Identity ({ address, identity }: Props): React.ReactElement<Props> | nu
   const { isRegistrar, registrars } = useRegistrars();
   const [isJudgementOpen, toggleIsJudgementOpen] = useToggle();
   const subs = useCall<[BalanceOf, AccountId[]]>(api.query.identity?.subsOf, [address])?.[1];
+  const [isOpen, toggleIsOpen] = useToggle();
```
```suggestion
  const [isSubsModalOpen, toggleSubsModalOpen] = useToggle();
```

MiZiet replied:
✅

krzysztof-jelski commented on packages/page-accounts/src/Sidebar/Sidebar.spec.tsx:
```diff
@@ -0,0 +1,128 @@
+    expect(subs).toHaveLength(0);
+  });
+
+  it('displays amount of subs', async () => {
```
```suggestion
  it('displays count of subs', async () => {
```
Count or number, amount goes better with money or uncountable nouns.

MiZiet replied:
✅

krzysztof-jelski commented:
For some accounts, the layout breaks:
![image](https://user-images.githubusercontent.com/430616/132668289-49db965a-4f1a-4a06-881c-2e53ddf52793.png)

krzysztof-jelski commented on packages/page-accounts/src/Sidebar/SubAccountsModal.tsx:
```diff
@@ -0,0 +1,69 @@
+      onClose={onClose}
+      size='tiny'
+    >
+      <Modal.Content className='content'>
```
It would be perfect to have just the content scrollable, with the header pinned to top. But perhaps that's a separate improvement to the new modal.

MiZiet replied:
I've set only content to be scrollable but only in this modal. But if it's needed we can make another pr with setting this to all modals.

MiZiet commented:
> For some accounts, the layout breaks:
> ![image](https://user-images.githubusercontent.com/430616/132668289-49db965a-4f1a-4a06-881c-2e53ddf52793.png)

I've set greater width there, it should be ok now.

MiZiet commented:
Styled multisig section

ekowalsk commented:
💪  Looks very good

MiZiet commented:
hi @jacogr, maybe you can give it a try and play with it?

this is how it looks like:
![image](https://user-images.githubusercontent.com/55240109/133052597-b08406ce-4eaa-4c42-9a66-3c9a54e1f108.png)
![image](https://user-images.githubusercontent.com/55240109/133052674-22ffa144-9944-4fe1-aa97-48c79865ca0b.png)

with current look we still have to click on dropdown to see all subs, which also causes some problem like jumping subs label or whole content jumping due to scrollbar appearing.

jacogr commented:
Have played through - the issue I have with it is that modal from modal is normally not a good practice. Sadly, it gets done in a couple (less than a handful) of places.

In this case, there is a work-around, so we don't need to wade into this approach. 

- we can expand sections
- we can scroll containers

MiZiet commented:
@jacogr so i've removed the subs modal and added scroll:

![image](https://user-images.githubusercontent.com/55240109/133266894-7565a203-1416-4326-a652-a74f68550871.png)

What do you think?

jacogr commented:
The above is fine, but would make it an expander - specifically for situations like these where it is just, well, a lot.

Since we merged the address styling, some conflicts snuck in here.

MiZiet commented:
hi @jacogr, I've resolved conflicts and wrapped subs into an expander (when their number is greater than the set threshold).  
What do you think?

jacogr reviewed (approved):
Thank you!
